src/styles/variables.scss

```
@@ -0,0 +1,9 @@
+$white: #fff;
+$main-color: #00acdc;
+$header-height: 60px;
+$link-color: #060b35;
+$primaryColor: #060b35;
```

No need to duplicate variables with the same values
```suggestion
$link-color: #060b35;
```